Dockerfile

```diff
@@ -14,8 +14,8 @@ COPY ./docker/php/php.ini /usr/local/etc/php/php.ini
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
 
-RUN pecl install xdebug \
- && docker-php-ext-enable xdebug
+RUN if [$PHP_VERSION == 8.0]; then pecl install xdebug \
```

Comment:
Kdyz uz delame stejne breaking changes & major release, nezvedneme rovnou min verzi PHP na 8.0 (8.1?) at tu nemusi byt workaroundy?

Reply:
thank you! yes! :-)

Reply:
this is library so use the minimal version it should support.. AFAIK we did not use 8.0 anywhere, skipped to 8.1 right away

Reply:
right, I kept the composer require at 8.0 so should be ok

Reply:
Well those two should be kept in sync :slightly_smiling_face: Composer check only that you run a supported version, but won't disallow you using 8.2 PHP featues in the code :)

Reply:
yeah, I see now that monolog 3.x requires ^8.1 so I'll switch it to that

Reply:
what a coincidence :wink: